images/ubuntu/scripts/build/configure-environment.sh

```diff
@@ -20,9 +20,11 @@ mkdir -p /etc/skel/.config/configstore
 setEtcEnvironmentVariable "XDG_CONFIG_HOME" '$HOME/.config'
 
 # Change waagent entries to use /mnt for swapfile
-sed -i 's/ResourceDisk.Format=n/ResourceDisk.Format=y/g' /etc/waagent.conf
-sed -i 's/ResourceDisk.EnableSwap=n/ResourceDisk.EnableSwap=y/g' /etc/waagent.conf
-sed -i 's/ResourceDisk.SwapSizeMB=0/ResourceDisk.SwapSizeMB=4096/g' /etc/waagent.conf
+if [[ "${SOURCE_TYPE}" == "azure-arm" ]]; then
```

Comment:
modified files are only available on azure image

Reply:
I think that it would be much better to have it as an individual post-build step (`provisioner`) running only in the case of `azure-arm.build_image`.